shared/common-adapters/relative-popup-hoc.desktop.js

```diff
@@ -223,7 +223,8 @@ function ModalPositionRelative<PP>(
 
     _handleScroll = throttle(
       () => {
-        this.props.onClosePopup()
+        // TODO?
+        // this.props.onClosePopup()
```

Comment:
seems to work better w/o this interaction. i'll likely remove this in a later branch